### ISSUE TITLE
Use search -> delete last gallery -> crash

### ISSUE THREAD

**Issue by Meowmel** [seanegoodwin/PandaViewer#71]
How to reproduce:
Pretty much as the title says: Search for an Artist, delete the last (read: the only one left, not the last from several ones) gallery,  Crash "An unhandled error occurred".
I just did a quick check with 3 different Artist and every time it crashed after the delete.

**Comment by Black-Light** [comment]
This might of been fixed in 0.9.1
"PandaViewer will now respond to changes on the filesystem while running (e.g. moving/renaming folders/files)."

Since we can't launch it however, who knows!

**Comment by seanegoodwin** [comment]
I don't think that fixed it. Although I haven't been able to work on this for a few months now, so who knows.

**Comment by Black-Light** [comment]
This still happens in 0.9.3 by the way.
